.github/CONTRIBUTING.md

```diff
@@ -22,6 +22,7 @@ Brave welcomes contributions via [pull requests](https://github.com/brave/browse
     Fix #206
 ````
 
+* If you update the npm dependencies, run `npm shrinkwrap --dev` before committing changes.
```

Comment:
Question: is it ok to run that everytime when I commit something to keep the dependencies up to date?